From f5ada0e7c332e9a317844e51fa385257be9faed9 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:10:16 -0400
Subject: [usb] option: Add ID for Telewell TW-LTE 4G v2

Message-id: <1410887463-20674-545-git-send-email-dzickus@redhat.com>
Patchwork-id: 95628
O-Subject: [RHEL7 PATCH 544/591] usb: option: Add ID for Telewell TW-LTE 4G v2
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 3d28bd840b2d3981cd28caf5fe1df38f1344dd60
Author: Bernd Wachter <bernd.wachter@jolla.com>
Date:   Wed Jul 2 12:36:48 2014 +0300

    usb: option: Add ID for Telewell TW-LTE 4G v2

    Add ID of the Telewell 4G v2 hardware to option driver to get legacy
    serial interface working

    Signed-off-by: Bernd Wachter <bernd.wachter@jolla.com>
    Cc: <stable@vger.kernel.org>
    Signed-off-by: Johan Hovold <johan@kernel.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/option.c b/drivers/usb/serial/option.c
index ac73f49..a968894 100644
--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -1487,6 +1487,8 @@ static const struct usb_device_id option_ids[] = {
   .driver_info = (kernel_ulong_t)&net_intf2_blacklist },
  { USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, 0x1426, 0xff, 0xff, 0xff),  /* ZTE MF91 */
   .driver_info = (kernel_ulong_t)&net_intf2_blacklist },
+ { USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, 0x1428, 0xff, 0xff, 0xff),  /* Telewell TW-LTE 4G v2 */
+  .driver_info = (kernel_ulong_t)&net_intf2_blacklist },
  { USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, 0x1533, 0xff, 0xff, 0xff) },
  { USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, 0x1534, 0xff, 0xff, 0xff) },
  { USB_DEVICE_AND_INTERFACE_INFO(ZTE_VENDOR_ID, 0x1535, 0xff, 0xff, 0xff) },
-- 
1.7.1